lib/Doctrine/Common/Annotations/AnnotationReader.php

```diff
@@ -325,7 +326,8 @@ private function getIgnoredAnnotationNames(ReflectionClass $class)
      */
     private function getClassImports(ReflectionClass $class)
     {
-        if (isset($this->imports[$name = $class->getName()])) {
+        $name = $class->getName();
```

Comment:
The expression was valid before - consider raising an issue in phpstan, since `$name` would have been defined in any case